generator/src/generator.rs

```diff
@@ -22,12 +22,17 @@ pub fn generate(
     generics: &Generics,
     path: &Path,
     rules: Vec<OptimizedRule>,
-    defaults: Vec<&str>
+    defaults: Vec<&str>,
+    include_grammar: bool,
```

Comment:
Would normally not use trailing commas.

Reply:
Habit from rustfmt style; I'll stick to the file's